payment/models.py

```diff
@@ -0,0 +1,19 @@
+from config.database import Base
+from sqlalchemy import Column, Integer, String,Time,DateTime, ForeignKey
+from sqlalchemy.orm import relationship
+import uuid
+from sqlalchemy.dialects.postgresql import UUID
+from datetime import datetime
+
+class Transaction(Base):
+    __tablename__ = "transaction"
+    id = Column(UUID(as_uuid=True),primary_key=True, default=uuid.uuid4,unique=True)
```

Comment:
I don't think using uuid as pk will work, use normal id as pk, then uuid as the id

Reply:
It works. The downside is the memory and query performance that can be optimized

Reply:
Okay